src/app/components/form/form.component.html

```diff
@@ -70,16 +70,15 @@ <h1>{{'Domain name'|translate}}</h1>
       <div *ngIf="is_advanced_options_enabled">
              <span [formGroup]="checkboxForm" class=" half-with">
               <span class="options d-flex" [formArrayName]="'items'" [class.invalid]="!checkboxForm['controls'].selectedItems.valid">
-                <span *ngFor="let protocol of checkboxForm['controls'].items['controls']; let i = index;"
+                <div *ngFor="let protocol of checkboxForm['controls'].items['controls']; let i = index;"
```

Comment:
Is it proper nowadays to have div-tags inside of span-tags?

Reply:
It is not ideal semantically speaking but would work without issue, I still revert it back to span